docker-compose.yaml

```diff
@@ -1,23 +1,33 @@
-db:
-    build: docker/test-db
+version: '2'
 
-mapserver:
-    build: docker/test-mapserver
-    links:
-    - db
+volumes:
 
-c2cgeoportal:
-    image: camptocamp/geomapfish_build_dev
-    links:
-    - db
-    - mapserver
-    volumes:
-    - build:/build
-    - .:/src
-    environment:
-        USER_NAME: ${USER_NAME}
-        USER_ID: ${USER_ID}
-        GROUP_ID: ${GROUP_ID}
-    stdin_open: true
-    tty: true
-    entrypoint: run ${RUN}
+    build: {}
+
+services:
+
+    db:
+        build: docker/test-db
+        image: camptocamp/c2cgeoportal-test-db
+
+    mapserver:
+        build: docker/test-mapserver
+        image: camptocamp/c2cgeoportal-test-mapserver
+        links:
+        - db
+
+    c2cgeoportal:
+        image: camptocamp/geomapfish_build_dev
+        links:
+        - db
+        - mapserver
+        volumes:
+        - build:/build
+        - .:/src
+        environment:
+            USER_NAME: ${USER_NAME}
+            USER_ID: ${USER_ID}
```

Comment:
You could use a hardcoded username within the container. Only the UID matters...